test/constants.js

```diff
@@ -0,0 +1,85 @@
+const Events = {
+    Success: "success",
```

Comment:
The following keys appear more than once in the object:

ResponseCode
Frozen
KycGranted
TokenExpiryInfo
TokenInfo
TokenKey
FungibleTokenInfo